### DISCUSSION TITLE
sonoff zigbee

### DISCUSSION THREAD

**Discussion by federico89mi** [indomus/forum#161] [Networking]
ciao a tutti,
vorrei avere delle vostre opinioni sui sonoff mini zigbee. Sono abbastanza inesperto ancora in questo campo, sto ristrutturando la mia casa e ho pensato di gestire tutte le luci di casa con i sonoff mini.. Vorrei optare per quelli nuovi zigbee perchè ho letto che il protocollo zigbee è molto più adatto alla domotica, consuma meno e le onde fanno meno male.. Voi cosa ne pensate?
Questi nuovi modelli zigbee sono validi ugualmente come quelli wifi ormai collaudati wifi?

Grazie a tutti

ciao

**Comment by Scialla** [comment]
vai su zigbee, tieni presente che questi non hanno la funziona "pulsante" ma solo "interruttore", quindi regolati

**Reply by snakuzzo** [reply]
In realtà "pare" funzionino anche con pulsante. Un utente in chat su Telegram che li ha li ha utilizzati sia con interruttore che con pulsante. Con pulsante pare che funzioni, ma dipende dal tempo di pressione. Devo fare una prova con quello che ho a casa

**Reply by federico89mi** [reply]
quindi sono differenti rispetto a quelli wifi?

**Reply by snakuzzo** [reply]
sì ... se sono zigbee sono sicuramente diversi da quelli wifi :D

**Reply by federico89mi** [reply]
il mio elettricista mi farà tutto con deviatori e invertitori.. vado tranquillo in questo caso giusto? Ma comunque me li consigliate rispetto ai wifi?

**Reply by federico89mi** [reply]
visto che devo praticamente fare tutto da zero.. volevo un consiglio su come farlo e cosa comprare

**Comment by KC-inDomus** [comment]
personalmente continuo ad esser dell'idea che gli attuatori non debbano mai esser scelti zigbee, ma wi-fi. l'affidabilità massima è ciò che conta, e il wi-fi in linea di massima - semplicemente perché ci sono meno attori nella catena operativa - è preferibile. solo sensori su zigbee, per quanto mi riguarda.

**Reply by Scialla** [reply]
però se ragioni così, devi andare a vedere se i sensori li usi per robe importanti, ad esempio per l'antifurto, oppure per gestire situazioni meteo pericolose; a sto punto tutto wifi... però poi tanti router hanno un limite di device, inoltre zigbee è mesh e i problemi di portata sono inferiori. 
Insomma, pro e contro, io voto zigbee :)

**Reply by snakuzzo** [reply]
Mio pensiero... la verità sta nel mezzo.
La scelta tra zigbee e wifi è soggettiva e non oggettiva. Dipende dalle singole situazioni e da quello che uno vuole.

E' sbagliato dire che Zigbee ha problemi di portata inferiori. E' vero che c'è il mesh tra i device, ma come ben sai non tutti i device fungono da router...quindi per avere il mesh devi piazzare router "alimentati" a destra e a manca...cosa che puoi fare tranquillamente anche con il WIFI aggiungendo dei repeater mesh (forse qui ti costa qualcosina in più)

Sul discorso "attori" nella catena operativa, non credo sia un problema avere un coordinator. Poi permettetemi di dire che la criticità legata alla tipologia di device è relativa allo scopo di utilizzo previsto dalla singola persona.

Lato Zigbee, almeno ad oggi, hai poco "potere". Con i device WFI oggi puoi fare il bello e cattivo tempo. Puoi customizzare praticamente qualsiasi comportamento, mentre per i device Zigbee sei ovviamente legato a quanto previsto da chi ha sviluppato il firmware per i device. Eccezione va fatta per casi in cui si sceglie la costruzione di device propri (come quelli con firmware PTVO).

Tornando alla domanda iniziale di @federico89mi...
personalmente, se avessi la disponibilità economica, per interruttori luci/tapparelle acquisterei tutti device Zigbee...ma di sicuro **non comprerei i sonoff mini**. Acquisterei attuatori già belli e pronti come serie Vimar o Bticino.

Se proprio vuoi restare su quella fascia di prezzo, a sto punto ti consiglierei di virare sul wifi e scegliere Shelly. Molto più comodi da piazzare nelle scatole 50x e soprattutto la gestione per le tapparelle (o simili) ad oggi è impareggiabile.

**Reply by KC-inDomus** [reply]
io non ho detto che abbia portata inferiore, ho solo detto - ed è oggettivo - che la catena operativa ZigBee è più lunga rispetto al Wi-Fi, e dato che l'affidabilità (IMHO) sugli attuatori dev'essere massima, preferisco quest'ultimo standard. in definitiva, un Sonoff lo comprerò sempre e solo Wi-Fi.
su Shelly invece sai che siamo perfettamente d'accordo, da sempre.

**Reply by snakuzzo** [reply]
Il discorso "portata" era relativo a quanto scritto da @Scialla :)

**Reply by KC-inDomus** [reply]
ah ok scusa, è che scialla non me lo cago :P

**Reply by Scialla** [reply]
Se hai problemi di stitichezza, e puoi aiutarti, vai a cagare! :D

**Reply by federico89mi** [reply]
a me il protocollo zigbee era stato "venduto" da dei miei conoscenti come protocollo per la domotica.. in più non occupa troppi ip nel router a differenza di quelli wifi e consuma meno.. di questo cosa ne pensate?

Se doveste fare casa da zero.. cosa usereste per interruttori luce e cosa usereste per le tapparelle?

**Reply by snakuzzo** [reply]
Sulla prima domanda... vero
Sulla seconda domanda ti ho già risposto [QUI](https://github.com/indomus/forum/discussions/161#discussioncomment-615003)

**Reply by federico89mi** [reply]
cosa sono questi attuatori già pronti @snakuzzo ? Che differenza di prezzo hanno con i sonoff per esempio?

**Reply by snakuzzo** [reply]
Beh...sì...costano tipo il triplo/quadruplo.
Prova a dare un'occhiata a Bticino e Vimar...li trovi anche su Leroy Merlin

**Reply by federico89mi** [reply]
no diventa troppo alto.. andata per i sonoff zigbee e mi faccio montare tutto dall'elettricista... Mi farà tutto con deviatori e invertitori.. che ne dite?

**Reply by snakuzzo** [reply]
Prima di partire "a cannone", fossi in te proverei il funzionamento a banco di un zbmini...anche per il discorso "pulsante".
Personalmente, non mi piace come scelta...credo sia la peggiore tra quelle possibili.
E personalmente credo che scegliere i zbmini al posto degli shelly sia follia

**Reply by federico89mi** [reply]
Nn capisco, qualè il discorso pulsante?

**Reply by snakuzzo** [reply]
che va visto bene se funzionano o meno. Funzionano al 100% con gli interruttori...ma è dichiarato NON funzionante con pulsanti.
Un utente dice che funzionano, ma dipende da QUANTO tieni premuto il pulsante. Io ne ho uno ma non ho avuto ancora modo di provare con un pulsante

**Reply by federico89mi** [reply]
Ma io come ti dicevo farò tutto con deviatore e invertitore che preferisco quindi non vedo problemi :)

**Reply by snakuzzo** [reply]
Sì hai ragione...non so dove li metterai, ma considera che in una classica 503 al massimo ci va un zbmini ed un interruttore/deviatore...non di più. Non si incastra nei tappi, ma devi posizionarlo in fondo alla scatola.
Secondo me è scomodo...però fai tu

**Reply by federico89mi** [reply]
Tutti nelle scatole d derivazione

**Reply by snakuzzo** [reply]
Ok allora

**Reply by KC-inDomus** [reply]
> a me il protocollo zigbee era stato "venduto" da dei miei conoscenti come protocollo per la domotica.. in più non occupa troppi ip nel router a differenza di quelli wifi e consuma meno.. di questo cosa ne pensate?
> 
> Se doveste fare casa da zero.. cosa usereste per interruttori luce e cosa usereste per le tapparelle?

ZigBee è UNO dei protocolli dedicati alla domotica. come detto prima, prevede una catena operativa più lunga - ergo ha un potenziale di fallibilità maggiore, per quanto sia tendenzialmente un ottimo standard. dato che sugli attuatori - per come la vedo io - non puoi permetterti di fallire, continuo ad essere dell'idea che per tali componenti il Wi-Fi sia tendenzialmente preferibile.
